premis_event_service/urls.py

```diff
@@ -1,21 +1,21 @@
-from django.conf.urls import url
+from django.urls import path, re_path
 from . import views
 
 urlpatterns = [
-    url(r'^APP/$', views.app, name='app'),
-    url(r'^APP/event/$', views.app_event, name='app-event'),
-    url(r'^APP/event/(?P<identifier>.+?)/$', views.app_event, name='app-event-detail'),
-    url(r'^APP/agent/$', views.app_agent, name='app-agent'),
-    url(r'^APP/agent/(?P<identifier>.+?)/$', views.app_agent, name='app-agent-detail'),
-    url(r'^event/$', views.recent_event_list, name='event-list'),
-    url(r'^event/search/$', views.event_search, name='event-search'),
-    url(r'^event/search.json$', views.json_event_search, name='event-search-json'),
-    url(r'^event/find/(?P<linked_identifier>.+?)/(?P<event_type>.+?)?/$',
-        views.findEvent, name='find-event'),
-    url(r'^event/(?P<identifier>.+?)/$', views.humanEvent, name='event-detail'),
-    url(r'^agent/$', views.humanAgent, name='agent-list'),
-    url(r'^agent/(?P<identifier>.+?).xml$', views.agentXML, name='agent-detail-xml'),
-    url(r'^agent/(?P<identifier>.+?).premis.xml$', views.agentXML, name='agent-detail-premis-xml'),
-    url(r'^agent/(?P<identifier>.+?).json$', views.json_agent, name='agent-detail-json'),
-    url(r'^agent/(?P<identifier>.+?)/$', views.humanAgent, name='agent-detail'),
+    path('APP/', views.app, name='app'),
+    path('APP/event/', views.app_event, name='app-event'),
+    path('APP/event/<identifier>/', views.app_event, name='app-event-detail'),
+    path('APP/agent/', views.app_agent, name='app-agent'),
+    path('APP/agent/<identifier>/', views.app_agent, name='app-agent-detail'),
+    path('event/', views.recent_event_list, name='event-list'),
+    path('event/search/', views.event_search, name='event-search'),
+    path('event/search.json', views.json_event_search, name='event-search-json'),
+    re_path('event/find/(?P<linked_identifier>.+?)/(?P<event_type>.+?)?/$',
```

Comment:
Yep, just ran some tests, and as-is this would match URLs beginning with `mainevent` or any other thing that just contains the pattern but has any preceding bits. So we should revert the string to what it was, that's how `re_path` wants it.